### PR TITLE
FreeRTOS Macro Update

### PR DESCRIPTION
**Pull request by UsefulElectronics**
In later FreeRTOS releases, the macro portTICK_RATE_MS was replaced with portTICK_PERIOD_MS.
All files were updated

### CODE REVIEW

File: main/.cproject
Comment (martinberlin) [inline main/.cproject:1]: - [ ] This files is from your IDE, they should be added in the .gitgnore, and should not be part of the repository

File: .cproject
Comment (martinberlin) [inline .cproject:1]: - [ ] This files is from your IDE, they should be added in the .gitgnore, and should not be part of the repository

File: .project
Comment (martinberlin) [inline .project:1]: - [ ] This files is from your IDE, they should be added in the .gitgnore, and should not be part of the repository

**Comment by martinberlin** [comment]
Hi @UsefulElectronics 
Do you think that this is the best solution or using pdMS_TO_TICKS(INT) would be better?

The only problem I see is that if there is still developers using ESP-IDF 4 to 4.2 then **portTICK_PERIOD_MS** will not be available, right?

File: main/.project
Comment (martinberlin) [inline main/.project:1]: - [ ] This files is from your IDE, they should be added in the .gitgnore, and should not be part of the repository
- [ ] Add .project in .gitignore
- [ ] Add .cproject in .gitignore (And any other IDE custom file)

**Review by martinberlin — CHANGES_REQUESTED** [review]
Please check the requested changes

**Comment by UsefulElectronics** [comment]
Both portTICK_PERIOD_MS and portTICK_RATE_MS were supported then.

Found portTICK_PERIOD_MS used in ESP-IDF V4.0 SDIO example.  
https://github.com/espressif/esp-idf/blob/release/v4.0/examples/peripherals/sdio/host/main/app_main.c

I will handle the requested changes thank you for the response.

I am also working on adding DEPG750BN Epaper.

**Review by martinberlin — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**